design/ceph/cephx-keyring-rotation.md

```diff
@@ -53,6 +90,13 @@ keyGeneration: <int> # used with KeyGeneration
     intended use case).
     If this is set to less than or equal to the current key generation, keys are not rotated.
 
+API for overlapping rotation:
+
+- `keepPriorKeyCount` (int): only available for components that use overlapping key rotation. This
```

Comment:
Are there any usecase or possibilities where Rook can remember multiple keys?

Reply:
We have designed this API to allow for any potential usecases. In theory, I can imagine someone rotating keys, getting halfway through node updates, and then deciding that they need to rotate keys again, and they might want to have 3 active keys in that case.